sql_generators/stable_views/__init__.py

```diff
@@ -23,6 +23,8 @@
 from bigquery_etl.dryrun import get_id_token
 from bigquery_etl.config import ConfigLoader
 
+BOT_GENERATED = 'LOWER(IFNULL(metadata.isp.name, "")) = "browserstack" AS bot_generated,'
```

Comment:
I wonder if we should move the `,` to be in the sql statement instead of the string here, if someone adds a new field later to the query they may miss the fact it's in the string already.
```suggestion
BOT_GENERATED = 'LOWER(IFNULL(metadata.isp.name, "")) = "browserstack" AS bot_generated'
```

Reply:
Yeah, I wasn't sure about the naming. `is_` is probably more indicative of a boolean flag